dev/io.openliberty.netty.internal.impl/test/io/openliberty/netty/internal/impl/NettyFrameworkImplTest.java

```diff
@@ -167,6 +168,91 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBuf buf) throws Excep
         Assert.assertTrue(framework.getActiveChannels().contains(testChannels.get(0)));
     }
 
+    /**
+     * Create an outbound UDP channel
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testUDPStartOutbound() throws Exception {
+        BootstrapExtended bootstrap = framework.createUDPBootstrapOutbound(null);
+        bootstrap.handler(new SimpleChannelInboundHandler<ByteBuf>() {
+            @Override
+            protected void channelRead0(ChannelHandlerContext ctx, ByteBuf buf) throws Exception {
+                // do nothing
+            }
+        });
+        final CountDownLatch latch = new CountDownLatch(1);
+        framework.startOutbound(bootstrap, "*", 0, future -> {
+            if (future.isSuccess()) {
+                testChannels.add(future.channel());
+                latch.countDown();
+            } else {
+                Assert.fail("framework failed to start: " + future.cause().getMessage());
+                latch.countDown();
+            }
+        });
+        framework.setServerStarted(null);
+        latch.await(PORT_BIND_TIMEOUT_MS, TimeUnit.MILLISECONDS);
+        Thread.sleep(10); // wait to ensure framework's internal callbacks are handled
+        Assert.assertTrue(testChannels.size() == 1);
+        Assert.assertTrue(testChannels.get(0).isActive());
+        Assert.assertTrue(framework.getActiveChannels().contains(testChannels.get(0)));
+    }
+
+    /**
+     * Create an inbound TCP channel, then create a new outbound TCP channel and
+     * connect to the inbound channel
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testTCPStartOutbound() throws Exception {
+        setTCPConfig();
+        ServerBootstrapExtended bootstrap = framework.createTCPBootstrap(options);
+        bootstrap.childHandler(bootstrap.getBaseInitializer());
+        final CountDownLatch latch = new CountDownLatch(1);
```

Comment:
I always find variables named like this a bit difficult to pass by on a review, here it does not really matter as it is used so nearby but I think it is a good habit to give it a descriptive name. Up to you of course as not a defect.

Reply:
I took a look and I agree it could be hard to follow up on but since this is just a unit test, I decided to leave as is for the time being. We can revisit this more in the future and make the updates to make this better to understand.